packages/crayons-core/src/components/country-phone/readme.md

```diff
@@ -0,0 +1,319 @@
+  hint-text="Input hint text"
+  warning-text="select country code to get valid phonenumber"
+  state="warning"
+></fw-country-phone>
+  <fw-country-phone
+  value="+1264497275926449727599878978789798798789798798789787897987897897897878978978787897989878"
+  name="country-code"
+  clear-input
+  select-placeholder="select country code"
+  input-placeholder="Enter phone number"
+  required
+  input-label="Support phone number"
+  select-label="Country Code"
+  hint-text="Input hint text"
+  error-text="Invalid Country Phone Number"
+  state="error"
+></fw-country-phone>
+```
+
+
+
+## Usage
+
+<code-group>
+<code-block title="HTML">
+```html
+<fw-country-phone
+    value=""
+    name="country-code"
+    clear-input
+    select-placeholder="select country code"
+    input-placeholder="Enter phone number"
+    required
+    input-label="Support phone number"
+    select-label="Country Code"
+    error-text="Error message text"
+    warning-text="Please use numbers for user ID"
+    state="normal"
+  ></fw-country-phone>
+<fw-country-phone
+    value="+919999999999"
+    name="country-code"
+    clear-input
+    select-placeholder="select country code"
+    input-placeholder="Enter phone number"
+    required
+    input-label="Support phone number"
+    select-label="Country Code"
+    error-text="Error message text"
+    warning-text="Please use numbers for user ID"
+    state="normal"
+    style="--fw-select-width: 0 0 400px"
+  ></fw-country-phone>
+  <fw-country-phone
+    value="+919999999999"
+    name="country-code"
+    clear-input
+    select-placeholder="select country code"
+    input-placeholder="Enter phone number"
+    required
+    input-label="Support phone number"
+    select-label="Country Code"
+    hint-text="select country code to get valid phonenumber"
+    warning-text="Please use numbers for user ID"
+    state="normal"
+  ></fw-country-phone>
+    <fw-country-phone
+  value="+1264497275926449727599878978789798798789798798789787897987897897897878978978787897989878"
+  name="country-code"
+  clear-input
+  select-placeholder="select country code"
+  input-placeholder="Enter phone number"
+  required
+  input-label="Support phone number"
+  select-label="Country Code"
+  hint-text="Input hint text"
+  warning-text="select country code to get valid phonenumber"
+  state="warning"
+></fw-country-phone>
+  <fw-country-phone
+  value="+1264497275926449727599878978789798798789798798789787897987897897897878978978787897989878"
+  name="country-code"
+  clear-input
+  select-placeholder="select country code"
+  input-placeholder="Enter phone number"
+  required
+  input-label="Support phone number"
+  select-label="Country Code"
+  hint-text="Input hint text"
+  error-text="Invalid Country Phone Number"
+  state="error"
+></fw-country-phone>
+```
+</code-block>
+
+<code-block title="React">
+```jsx
+import React from "react";
+import ReactDOM from "react-dom";
+import { FwInput } from "@freshworks/crayons/react";
```

Comment:
Kindly test this out before before adding to documentation. We need to import 'FwCountryPhone'.

Reply:
Tested in React Project & updated doc ✅